src/Sulu/Bundle/ContentBundle/Tests/Functional/Controller/NodeControllerTest.php

```diff
@@ -801,11 +811,30 @@ public function testPutState()
         $this->assertEquals(1, $response['nodeState']);
     }
 
+    private function import($file)
+    {
+        if ($this->session->getRootNode()->hasNode('cmf')) {
+            $this->session->getNode('/cmf')->remove();
+            $this->session->save();
+        }
+
+        $this->session->importXML(
+            '/',
+            __DIR__ . '/../../app/Resources/exports/' . $file . '.xml',
```

Comment:
Would it? Would make only sense if there might be a reason for it to change. But if we include another file here, the tests will probably fail anyway.